app/boards/shields/nice_view_adapter/README.md

```diff
@@ -0,0 +1,11 @@
+# nice!view Adapter
+
+This shield is used as an adapter between the nice!view and existing shields/boards that expose an I2C OLED header.
+
+To use this shield, you should add this shield to your list of shields *before* `nice_view`.
+
+The nice!view will use the SDA/SCL pins of the OLED, and then the adapter expects a final pin to be "bodged" from your microcontroller to the nice!view CS pin. This adapter assumes that the CS pin bodged is the `&pro_micro 1 pin` or "D1", which is the top left pin when looking at the front of the board. If you can't use this pin, you'll need to write your own definition for the `&nice_view_spi` bus.
```

Comment:
```suggestion
The nice!view will use the SDA/SCL pins of the header, and then the adapter expects a final pin to be "bodged" from your microcontroller to the nice!view CS pin. This adapter assumes that the CS pin bodged is the `&pro_micro 1 pin` or "D1", which is the top left pin when looking at the front of the board. If you can't use this pin, you'll need to override the `cs-gpios` definition for the `&nice_view_spi` bus.
```
The first sentence might be confusing because there is actually no OLED display involved here. Also thoughts on making it more specific as such for the last sentence?

Reply:
Agreed on the first sentence. Second sentence... I think the override is only possible on zmk-config keymaps? The reason being is we need the override to be the last DT override, and I think currently that's only possible with the zmk-config if I'm not mistaken.

Reply:
I might be misunderstanding, but I believe you always need it to be defined last, right? Even if you create a new `nice_view_spi` node instead of overriding with `&nice_view_spi`, it still needs to be defined later for it to take effect instead of the node definition coming from board-specific overlays.